app/src/main/java/com/example/opticcommunication/decoder/DecoderRC5.java

```diff
@@ -0,0 +1,24 @@
+package com.example.opticcommunication.decoder;
+
+import java.nio.charset.StandardCharsets;
+
+public class DecoderRC5 {
+
+    private static final int frameLength = 12;
+    private DecoderRC5() {}
```

Comment:
Private constructor?

Reply:
mhm, don't want to allow for creating instance of the class